docker/docker-compose.integration-scylladb.yml

```diff
@@ -0,0 +1,15 @@
+services:
+  scylladb_example:
+    image: bitnami/scylladb:6.0.0
```

Comment:
See comments on bitnami image in corresponding plus PR! https://github.com/ethyca/fidesplus/pull/1447#discussion_r1636464528